### ISSUE TITLE
Compatibility with "Adaptive Notation"

### ISSUE THREAD

**Issue by cyberphone** [mikunn/openapi-schema-to-json-schema#14]
To cope with the IEEE-754 double precision limit the Java folks use what I call *Adaptive Notation* where the choice between using JSON Number and putting the value within a JSON String is based on the actual value itself:
```json
{
  "BigDecimal.large":"1E+999",
  "BigDecimal.small":1,
  "BigInteger.large":"1787787787877878787878787878787",
  "BigInteger.small":1,
  "Long.large":"9223372036854775807",
  "Long.small":1
}
```
How does this work in an OpenAPI/JSON Schema setting?

**Comment by ehmicky** [comment]
I do not think this is something what is covered by either OpenAPI nor JSON schema, and in that respect, is beyond the scope of this library.
